src/server.js

```
@@ -54,6 +54,16 @@ app.use(expressJwt({
   credentialsRequired: false,
   getToken: req => req.cookies.id_token,
 }));
+// Error handler for express-jwt
+app.use((err, req, res, next) => { // eslint-disable-line no-unused-vars
+  if (err instanceof Jwt401Error) {
+    console.error('[express-jwt-error]', req.cookies.id_token);
+    // `clearCookie`, otherwise user can't use web-app until cookie expires
+    res.clearCookie('id_token');
+  }
+  next(err);
+});
+
```

So the error is swallowed, I'd rather present it in errorPage.
If someone want it to be swallowed like the code I wrote in first commit, he can change it easily.
But if we swallow it directly, he might won't know there's an error at all.